apps/base-docs/tutorials/docs/6_subquery.md

```diff
@@ -0,0 +1,364 @@
+---
+title: 'Indexing Data with SubQuery'
+slug: /subquery
+description: A tutorial explaining SubQuery and providing swift guidance on configuring a project
+author: subquery
+keywords:
+  [
+    indexers,
+    RPC,
+    API,
+    graphql,
+    frontend,
+    smart contract development,
+    EVM,
+    Base,
+    Base network,
+    Base node providers,
+    Base providers,
+    blockchain development,
+    dApps,
+    smart contracts,
+    blockchain API
+  ]
+tags: ['smart contracts', 'api']
+difficulty: beginner
+hide_table_of_contents: false
+displayed_sidebar: null
+---
+
+## Objectives
+
+- Present the concept of indexers
+- Begin effectively with all types of SubQuery indexer configuration files
+- Successfully build your project
+- Efficiently retrieve data from indexed sources
+
+The goal is to index the all the claims from the [Bridge to Base NFT contract](https://basescan.org/token/0xEa2a41c02fA86A4901826615F9796e603C6a4491) on [Base Mainnet](https://docs.base.org/using-base/).
+
+---
+
+:::info
+
+The final code of this project can be found [here](https://github.com/subquery/ethereum-subql-starter/tree/main/Base/base-nft).
+
+:::
+
+---
+
+The contents presented in this tutorial were discussed previously during the workshop. You can follow along to receive detailed audio commentary on the setup's intricacies:
+
+<iframe width="1317" height="741" src="https://www.youtube.com/embed/8N5h3SI8E-A" title="Base x SubQuery Indexing Workshop" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture; web-share" referrerpolicy="strict-origin-when-cross-origin" allowfullscreen></iframe>
+
+## What are indexers?
+
+Indexers, in a broad context, play a fundamental role in organising and optimising data retrieval within various systems. These tools act as navigational aids, allowing efficient access to specific information by creating structured indexes. 
+
+In the context of blockchain and dApps, indexers go beyond traditional databases, facilitating streamlined access to on-chain data. This includes transaction histories, smart contract states, and event logs.
+
+## Setup
+
+In this chapter, we will delve into the intricacies of SubQuery indexer configuration. The chapter is subdivided into sections dedicated to each specific file type, all of which require configuration.
```

Comment:
Please rewrite to talk to the learner with "you" instead of we.  More detail in the style guide:  https://github.com/base-org/web/wiki/Style-Guide-for-Tutorials